### PR TITLE
Contact form added.

### PR DESCRIPTION
**Pull request by DilsherB**
Contact form added.

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi @DilsherB , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has non exhaustive README file and a bad PR decription, kindly update your README file with all the necessary fields from the template you used and also please consider writing a meaning and descriptive PR description with title. The submit again for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by roshan-bajgain — COMMENTED** [review]
Hi @DilsherB, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has been done alone but this is a pair-programming project and we will check the commit history. Please implement the project with your partner.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏a

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -137,6 +137,19 @@ <h2 id="introHeader">About Myself</h2>
...
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
+            <ul>
+                <li><input type="text" name="name" id="name" placeholder="Dilsher" class="tb" maxlength="30" required></li>
+                <li><input type="email" name="email" id="email" placeholder="dilsherb@gmail.com" class="tb" required></li>
+            </ul>
+            <textarea name="msg" name="msg" id="msg" cols="30" rows="10" placeholder="Write your message here." maxlength="500" required></textarea>
+            <button type="submit" id="submit">Get in touch</button>
+        </form>
+    </section>
```
Comment (TadesseDev) [inline index.html]: - [ ] Good job adding the `action` and `method` attributes, But currently user cannot submit to `formspree.io`. As you can see down from the browser dev tools your form spare is not found. Please make sure the user can successfully submit a form to  `formspree.io`, Without this feature the form will be useless.
Capture

File: style.css [hunk 2]
```diff
@@ -19,6 +19,12 @@ button {
   padding: 12px;
 }
 
+form > button {
+  font-weight: 500;
+  width: 150px;
+  font-size: 17px;
+}
```
Comment (TadesseDev) [inline style.css]: ### Good job on the CSS @DilsherB, Some suggestion will be 
- [ ] Please use `flexbox` to align elements here, This is among the main requirements. 
- [ ] Please make sure `your form content` is aligned center and the button is aligned to the right as displayed in the Figma design. 
- [ ] Please apply the appropriate background images for the form. This is to make sure you are capable of parsing a design into a webpage. 
> Remember: you are expected to stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding).

### Figma design
figma

### Your design
your

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @DilsherB 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
EquatorialVictoriousBlackpanther-size_restricted

## Highlights
 - [ ] Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (DilsherB) [reply style.css]: @Tadesse-Alemayehu

Comment (DilsherB) [reply style.css]: thanks for your kind review all the points raised by you are resolved

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @DilsherB 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**